src/export/digitalPlanning/model.ts

```diff
@@ -836,6 +838,7 @@ export class DigitalPlanning {
     const ldcDescriptionFns = [
       this.passport.data?.["proposal.buildingOperations.details"] as string,
       this.passport.data?.["proposal.changeOfUse.details"] as string,
+      this.passport.data?.["proposal.existingBreach.details"] as string,
```

Comment:
This change comes from August here https://opensystemslab.slack.com/archives/C089LL9LR5H/p1741017668220829